Armadillo/src/main/java/com/scribd/armadillo/StateStore.kt

```diff
@@ -25,7 +27,7 @@ internal interface StateStore {
     }
 }
 
-internal class ArmadilloStateStore(private val reducer: Reducer) :
+internal class ArmadilloStateStore(private val reducer: Reducer, private val appContext: Context) :
```

Comment:
Do we have to worry about this leaking the context now that it's holding a reference to it?

Reply:
Would it help if we held a reference to the looper instead? `ArmadilloStateStore(..., appContext: Context) ... private val looper = appContext.mainLooper`

Reply:
This is the app context rather than the activity context, so it shouldn't be a danger for leaks the way `Activity` is

Reply:
Resolved by passing in Handler